### DISCUSSION TITLE
discussion/index

### DISCUSSION THREAD

**Discussion by giscus** [arXiv/discussion#11] [Announcements]
# discussion/index

https://arxiv.github.io/discussion/index.html

**Comment by cbf66** [comment]
This is a community comment!

**Comment by bdc34** [comment]
What if I make a comment via the github web ui?

**Reply by bdc34** [reply]
Ya, you can see that in both github and giscus.

**Reply by cbf66** [reply]
Of course. Ultimately the backing store of giscuss is GitHub Discussions.
